### PR TITLE
[irods/irods#7503] iadmin: Enhance get/set_grid_configuration help text (main)

### PR DESCRIPTION
**Pull request by alanking**
Addresses irods/irods#7503

Also added the available namespaces/options for each command so you don't have to go look it up in the catalog.

### CODE REVIEW

File: src/iadmin.cpp [hunk 1]
```diff
@@ -2675,6 +2677,13 @@ usage( char *subOpt ) {
         "Set the specified OPTION_NAME to OPTION_VALUE for the specified",
         "NAMESPACE in the local zone in R_GRID_CONFIGURATION. If NAMESPACE or",
         "OPTION_NAME is not valid, an error is returned.",
+        " ",
+        "The grid configuration NAMESPACEs and their respective OPTION_NAMEs for",
+        "set_grid_configuration are:",
+        "    authentication",
+        "        password_extend_lifetime",
+        "        password_max_time",
+        "        password_min_time",
```
Comment (korydraughn) [inline src/iadmin.cpp:2686]: Do you feel users will understand this structure?
Should we provide an example or explain the structure somewhere so people know how to use it?

**Review by korydraughn — COMMENTED** [review]
Looks good. Just had one question.

File: src/iadmin.cpp (hunk 1 above)
Comment (alanking) [reply src/iadmin.cpp:2686]: Can always just make a table, I suppose...
```
NAMESPACE             OPTION_NAME
authentication        password_extend_lifetime
authentication        password_max_time
authentication        password_min_time
```

Comment (trel) [reply src/iadmin.cpp:2686]: much easier to read / understand i think - yes.

File: src/iadmin.cpp [hunk 2]
```diff
@@ -2675,6 +2677,13 @@ usage( char *subOpt ) {
         "Set the specified OPTION_NAME to OPTION_VALUE for the specified",
         "NAMESPACE in the local zone in R_GRID_CONFIGURATION. If NAMESPACE or",
         "OPTION_NAME is not valid, an error is returned.",
+        " ",
+        "The grid configuration NAMESPACEs and their respective OPTION_NAMEs for",
+        "set_grid_configuration are:",
```
Comment (trel) [inline src/iadmin.cpp:2682]: "are" -> "are currently"

File: src/iadmin.cpp [hunk 3]
```diff
@@ -2689,6 +2698,19 @@ usage( char *subOpt ) {
         " ",
         "This information is retrieved from the R_GRID_CONFIGURATION database",
         "table.",
+        " ",
+        "The grid configuration NAMESPACEs and their respective OPTION_NAMEs for",
+        "get_grid_configuration are:",
```
Comment (trel) [inline src/iadmin.cpp:2703]: same

File: src/iadmin.cpp (hunk 1 above)
Comment (alanking) [reply src/iadmin.cpp]: Done.

File: src/iadmin.cpp (hunk 2 above)
Comment (alanking) [reply src/iadmin.cpp]: Done

File: src/iadmin.cpp (hunk 3 above)
Comment (alanking) [reply src/iadmin.cpp]: Done

**Comment by trel** [comment]
looking good.   ready for squashing if no other addendums.

File: src/iadmin.cpp (hunk 1 above)
Comment (korydraughn) [reply src/iadmin.cpp]: > much easier to read / understand i think - yes.

I agree.

**Review by korydraughn — APPROVED** [review]
Looks ready to me.

Squash/Pound when ready.

**Comment by alanking** [comment]
Done, thanks

**Pull request merged**